package/src/types/raw/competitive-updates.ts

```diff
@@ -0,0 +1,23 @@
+import { Region } from "../general"
+
+interface Match {
+	MatchID: string
```

Comment:
There is still a tab indentation instead of spaces, I will just comment this here, but there will probably be more code parts

Reply:
The problem was that eslint doesn't recognize interfaces on it's own, so I'm using the plugin `typescript-eslint` which has its own separate indent rule that extends off of eslint's one. I was still using the vanilla indent rule, my fault